dart-runtime/lib/deviceinfo_web.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/widgets.dart';
+import 'dart:html';
+
+Future<Map<String, Object?>> getDeviceInfo(
+    BuildContext? context, String deviceId) async {
+  var locale = context == null ? null : Localizations.localeOf(context);
+
+  return {
+    "id": deviceId,
+    "language":
+        locale == null ? null : "${locale.languageCode}-${locale.countryCode}",
+    "platform": {"browserUserAgent": window.navigator.userAgent},
+    "timezone": DateTime.now().timeZoneName,
```

Comment:
This is going to deviate from the current sdkgen standard ([TZ Database Ids](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), like `America/Sao_Paulo`). Instead, it can send strings like `Brasilia Standard Time`. We can use the [`timezone` library](https://pub.dev/packages/timezone) to avoid that, but it increases the app size

Reply:
`DateTime.now().timeZoneName` is clearly broken, but the [timezone](https://pub.dev/packages/timezone) library doesn't seems to help with that. Given a name, it fetches information about that zone, but I couldn't find a reliable way to get the local time zone name.

I guess this should just stay as is for now, until we find a better sollution. It was already broken before this PR, I'm not making it worse :D